tests/sentry/grouping/test_enhancer.py

```diff
@@ -105,7 +106,7 @@ def assert_no_matching_frame_found(
 
 
 @pytest.mark.parametrize("version", [3])
-def test_basic_parsing(insta_snapshot, version):
+def test_basic_parsing(insta_snapshot: InstaSnapshotter, version: int | None) -> None:
```

Comment:
Shouldn't `version` just be an int? I know that the place it's getting plugged into can take `None`, but the test itself shouldn't.